db/seed_helper.rb

```diff
@@ -1,16 +1,14 @@
 require 'open-uri'
 require 'json'
-# require 'dotenv'
-# Dotenv.load('.env')
+require 'dotenv'
```

Comment:
Please comment out lines 3 and 5. I found out that these don't run in production.

Reply:
fixed in [1236af3](https://github.com/chrononaut76/rails-shopwise/pull/108/commits/1236af3a7c9e3f399bb83c64ab00e8b6747532f6)